features/users/github_commit_count.feature

```diff
@@ -19,7 +19,7 @@ Feature: Displaying GitHub contribution statistics for user
   Scenario: Displays commit counts on user profile page
     Given I am on "profile" page for user "Bryan"
     Then I should see "Contributions"
-    And I should see "WebsiteOne - 388"
+    And I should see "WebsiteOne - 394"
```

Comment:
Just a something to keep in mind here. see my commit message https://github.com/diraulo/WebsiteOne/commit/f3565d04b023ca275894b10d8d8ac2da385c5be7 for more details